hack/make-release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# set vars 
+if [ -n "$1" ]; then
+    DOCKER_REPO=$1
+else
+    echo "Must provide a docker repo"
```

Comment:
1. log to stderr
2. exit with error code


I recommend you create 2 funcs for logging stuff


log() { echo "$*" >&2 }
fail() { log "$*"; exit 1 }